src/test/compile-fail/issue-77910-1.rs

```diff
@@ -0,0 +1,11 @@
+fn foo(s: &i32) -> &i32 {
```

Comment:
oh, instead of `compile-fail` please move this test into `test/ui/binop` instead.

Reply:
Ah, my fault.